dotnet/flatpak-dotnet-generator.py

```diff
@@ -33,9 +37,9 @@ def main():
             'flatpak', 'run',
             '--env=DOTNET_CLI_TELEMETRY_OPTOUT=true',
             '--env=DOTNET_SKIP_FIRST_TIME_EXPERIENCE=true',
-            '--command=sh', '--runtime=org.freedesktop.Sdk//21.08', '--share=network',
-            '--filesystem=host', 'org.freedesktop.Sdk.Extension.dotnet6//21.08', '-c',
-            'PATH="${PATH}:/usr/lib/sdk/dotnet6/bin" LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/lib/sdk/dotnet6/lib" exec dotnet restore "$@"',
+            '--command=sh', '--runtime=org.freedesktop.Sdk//{}'.format(args.freedesktop), '--share=network',
+            '--filesystem=host', 'org.freedesktop.Sdk.Extension.dotnet{}//{}'.format(args.dotnet, args.freedesktop), '-c',
+            'PATH="${PATH}:'+'/usr/lib/sdk/dotnet{0}/bin" LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/usr/lib/sdk/dotnet{0}/lib" exec dotnet restore "$@"'.format(args.dotnet),
```

Comment:
Should be able to use f-strings for all of these (also using `${{PATH}}` to escape the brackets is cleaner than putting it in a separate string imo)

Reply:
Good point. Haven't really written python since py2.6.. Goes to show the times have changed.

Reply:
fixed as of 942b8e7.